UnicornRender/video/include/unicorn/video/Renderer.hpp

```diff
@@ -68,30 +69,29 @@ class Renderer
      */
     UNICORN_EXPORT virtual void SetDepthTest(bool enabled) = 0;
 
-    /** @brief  Creates new geometry mesh
-    *
-    *  Creates and subscribes to mesh.
-    *  Mesh shall be deleted via DeleteMesh().
-    *
-    *  @param [in] material describes mesh visual representation
-    *
-    *  @attention  Mesh lifetime is bound by its renderer's lifetime.
-    *              Using meshes after their renderer was destroyed is undefined behaviour.
-    *              If you're storing mesh pointers, consider storing them with a reference
-    *              to their renderer and listening for its Destroyed event for proper cleanup.
-    *
-    *  @return pointer to newly created geometry::Mesh
-    *
-    *  @sa DeleteMesh
+    /**
+    * @brief Adds mesh to rendering system
+    * @param [in] mesh mesh data
+    */
+    UNICORN_EXPORT virtual void AddMesh(Mesh* mesh) = 0;
+
+    /**
+    * @brief Adds multiple meshes simultaneously to rendering system
+    * @param [in] meshes meshes data
     */
-    UNICORN_EXPORT virtual Mesh* SpawnMesh(Material const& material) = 0;
+    UNICORN_EXPORT virtual void AddMeshes(std::list<Mesh*> const& meshes) = 0;
```

Comment:
Would it make more sense to just provide iterator range or is there more functionality planned than a simple loop?